### PR TITLE
Update for Ruby 3 support

### PR DESCRIPTION
**Pull request by luispadron**
Updates the files and use of positional/keyword arguments to support Ruby 3 update

### CODE REVIEW

**Review by justinseanmartin — APPROVED** [review]

**Comment by luispadron** [comment]
Tested upstream gem can depend on this correctly and use it in Ruby 3

File: lib/chronometer/dsl.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def method(method_name, **opts)
       opts[:method] = method_name
       opts[:event_type] ||= :X
       opts[:cls] = @cls || raise('must be in for_class block')
-      events << Event.new(opts)
+      events << Event.new(**opts)
```
Comment (luispadron) [inline lib/chronometer/dsl.rb:31]: This was the real change we needed https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

**Comment by luispadron** [comment]
Thanks @segiddins!

**Comment by luispadron** [comment]
@segiddins Could you release a new version when you get a chance 🙏🏼

**Pull request merged**